SixTrack/naff/windows.cc

```diff
@@ -117,7 +116,7 @@ std::vector<std::complex<double>> taylor_window(const size_t N, const double SLL
   std::vector<std::complex<double>> out;
   std::vector<double> ww(N);
   double A = acosh(pow(10, SLL)) / pi;
-  double NBAR = boost::math::round((2 * pow(A, 2) + 0.5)+0.5);
```

Comment:
Cool!

One request, could you name it as something else than just `round`, rather `boost_round` or something like that? There is so much stuff going on with rounding in this code already, let's not confuse things (or confuse the linker)...

For the cmath stuff, the fact that these are overloaded C++ functions should make sure they are name-mangled and therefore unique at compile time.